src/seedu/addressbook/commands/SortCommand.java

```diff
@@ -0,0 +1,21 @@
+package seedu.addressbook.commands;
+import seedu.addressbook.data.person.ReadOnlyPerson;
+import java.util.List;
+
+/**
+ * Sort contact list of the addressbook
```

Comment:
Good documentation before the class implementation to have a overview of what the class will be doing.